src/blocks/GroupingBlock/GroupingBlock.test.ts

```diff
@@ -0,0 +1,242 @@
+import { renderToFragment } from '@lib/renderer';
+import { describe, expect, test } from 'vitest';
+import GroupingBlock, { type Props } from './GroupingBlock.astro';
+
+describe('GroupingBlock', () => {
+  test('renders a stack layout without a title when layout is "stack-untitled"', async () => {
```

Comment:
Same layout tests as done previously on Page Partial

Reply:
Let's not duplicate it. Remove the PagePartial tests that are about rendering it with a certain layout, and let the PagePartialBlock use the GroupingBlock to render.